src/App.js

```diff
@@ -1,23 +1,92 @@
-import logo from './logo.svg';
-import './App.css';
+import { useState } from "react";
+import { useSelector } from "react-redux";
+import OverviewCards from "./components/OverviewCards";
+import ResourcesSection from "./components/ResourcesSection";
+import AddResourceForm from "./components/AddResourceForm";
+import "./main.css";
+import { selectResources } from "./store/resources/selectors";
+import {
+  favoritesWithThisDeveloper,
+  selectLoggedinUser,
+} from "./store/selectors";
+import {
+  selectDevelopers,
+  developersWithThisFavorite,
+} from "./store/developers/selectors";
 
 function App() {
+  const [resource, setResource] = useState(null);
+  const [developer, setDeveloper] = useState(null);
+  const resources = useSelector(selectResources);
+  const developers = useSelector(selectDevelopers);
+  const user = useSelector(selectLoggedinUser);
+
+  const resId = resource && resources.find((tool) => tool.name === resource).id;
```

Comment:
Since you're using the new syntax already somewhere else, these can all be ```resource?.find...``` and you skip the ```&&```s